sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -311,6 +322,31 @@ class HadoopTableReader(
     // Only take the value (skip the key) because Hive works only with values.
     rdd.map(_._2)
   }
+
+  /**
+   * Creates a NewHadoopRDD based on the broadcasted HiveConf and other job properties that will be
+   * applied locally on each slave.
+   */
+  private def createNewHadoopRdd(
+      tableDesc: TableDesc, path: String, inputClassName: String): RDD[Writable] = {
```

Comment:
nit: 
```
private def createNewHadoopRdd(
      tableDesc: TableDesc,
      path: String,
      inputClassName: String): RDD[Writable] = {
```

Reply:
Maybe it's also okay when a method declaration parameters fit two lines?

Reply:
Done, because of removing `inputClassName: String`.